docs/plugins/convert.rst

```diff
@@ -48,6 +48,12 @@ To test your configuration without taking any actions, use the ``--pretend``
 flag. The plugin will print out the commands it will run instead of executing
 them.
 
+By default, files that do not need to be transcoded will be copied to their
+destination. Passing the ``-l`` (``--link``) flag creates symbolic links
+instead, passing ``-H`` (``--hardlink``) creates hard links.
+Refer to the ``link`` and ``hardlink`` options below
+for potential issues with this.
```

Comment:
Maybe it would be nice to mention exactly what those issues are—namely, just say that album art embedding won't work?